sdk/communication/azure-communication-callingserver/src/test/java/com/azure/communication/callingserver/CallingServerTestBase.java

```diff
@@ -109,7 +112,17 @@ protected String getGroupId(String testName) {
           have unique groupId's so they do not conflict with other
           recording tests running in live mode.
          */
-        if (getTestMode() == TestMode.LIVE) {
+        if (getTestMode() == TestMode.LIVE) {        
+            return UUID.randomUUID().toString();
+        }
+
+        /*
+          The Live test pipeline runs in recording mode, so we 
```

Comment:
This isn't true, the live test pipeline runs in LIVE mode. What is the expected behaviour that you are trying to accomplish?

Reply:
Ours seems to run in Record, maybe it is configured incorrectly

Reply:
[Link to Live Pipeline test run](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=960406&view=logs&j=186fb226-6090-5bc4-037b-dfee8c37ef57&t=24176779-b866-57d9-80d3-8324e431459f&l=19)

Reply:
Sorry I just double checked, you are right, it is in record mode.

Reply:
I just talked with Alan Zimmer, and it seems the tests were temporarily switched to RECORD because LIVE was causing issues in azure-core-tests. This is going to be changed back soon. I will look at removing this work around for our tests, and also forcing LIVE mode for these tests, just incase there is another change.

Reply:
Sounds good, thanks for clarifying.